code/game/objects/items/weapons/implants/implantfluff.dm

```diff
@@ -0,0 +1,114 @@
+
+/**
+ * # Implant Fluff Datum
+ *
+ * All implants have used to have a get_data() call which used to return html, this datum serves to better method to
```

Comment:
nobody cares about how terrible the system used to be when reading doc comments

Reply:
removed